### PR TITLE
staging-next 2023-03-04

### PR DESCRIPTION
**Pull request by mweinelt**
##### Helpful links

- workflow docs: https://nixos.org/manual/nixpkgs/unstable/#submitting-changes-commit-policy
- important jobs:
  https://hydra.nixos.org/job/nixpkgs/staging-next/unstable#tabs-constituents
  https://hydra.nixos.org/job/nixos/staging-next-small/tested
- jobsets:
  https://hydra.nixos.org/jobset/nixpkgs/staging-next
  https://hydra.nixos.org/jobset/nixos/staging-next-small
- nix-review reports: https://malob.github.io/nix-review-tools-reports/
- previous staging-next: https://github.com/NixOS/nixpkgs/pull/216499

- - - 
##### Mass breakages

- [x]  `openjdk`, `jdk11`, `jdk17` fail to mv files in installPhase on aarch64-linux, e.g., https://hydra.nixos.org/build/211417115, https://github.com/NixOS/nixpkgs/issues/219513, bisected to 87bfed87336d693fcf2de10c9a1861ea4d6ca6ae @trofi
 - [x] nftables checked rules fails on aarch64-linux. `liblkl-hijack.so: undefined symbol: __aarch64_ldadd4_sync`, 
 https://hydra.nixos.org/build/211742721, https://hydra.nixos.org/build/211742732 @yu-re-ka

### CODE REVIEW

**Comment by vcunat** [comment]
The `python3*Packages.aiohttp` tests have been flaky lately.

**Comment by mweinelt** [comment]
Forgot to squash, sorry.

**Comment by doronbehar** [comment]
Merge conflict in `pkgs/development/python-modules/dbus-python-client-gen/default.nix`

**Comment by kirillrdy** [comment]
fix for awscli2 https://github.com/NixOS/nixpkgs/pull/221210

File: pkgs/development/python-modules/watchdog/default.nix [hunk 1]
```diff
@@ -32,37 +32,62 @@ buildPythonPackage rec {
...
+    "--deselect=tests/test_fsevents.py::test_watcher_deletion_while_receiving_events_1"
+    "--deselect=tests/test_fsevents.py::test_watcher_deletion_while_receiving_events_2"
+    "--deselect=tests/test_skip_repeats_queue.py::test_eventlet_monkey_patching"
+    "--deselect=tests/test_fsevents.py::test_recursive_check_accepts_relative_paths"
+    # fsevents:fsevents.py:318 Unhandled exception in FSEventsEmitter
+    "--deselect=tests/test_fsevents.py::test_watchdog_recursive"
+    # SystemError: Cannot start fsevents stream. Use a kqueue or polling observer...
+    "--deselect=tests/test_fsevents.py::test_add_watch_twice"
+    # fsevents:fsevents.py:318 Unhandled exception in FSEventsEmitter
+    "--deselect=ests/test_fsevents.py::test_recursive_check_accepts_relative_paths"
```
Comment (ShamrockLee) [inline pkgs/development/python-modules/watchdog/default.nix:81]: There seems to be a typo in the path. Is this deselection effective?

Comment (mweinelt) [reply pkgs/development/python-modules/watchdog/default.nix:81]: Unlikely

Comment (ShamrockLee) [reply pkgs/development/python-modules/watchdog/default.nix:81]: Addressed in PR #424799. Please take a look.

**Pull request merged**